src/DynamoCoreWpf/Views/Core/NodeView.xaml

```diff
@@ -23,7 +23,7 @@
 
         <Grid.RowDefinitions>
             <RowDefinition Height="Auto"></RowDefinition>
-            <RowDefinition Height="Auto"></RowDefinition>
+            <RowDefinition Height="Auto" MaxHeight="27"></RowDefinition>
```

Comment:
When a node with custom content is dragged, so that its height much longer than default, its nickname rectangle has bigger height than its origin node does. So I fixed height